ambry-mysql/src/integration-test/java/com/github/ambry/accountstats/AccountStatsMySqlStoreIntegrationTest.java

```diff
@@ -499,6 +521,9 @@ private AccountStatsMySqlStore createAccountStatsMySqlStore(String clusterName,
     configProps.setProperty(AccountStatsMySqlConfig.UPDATE_BATCH_SIZE, String.valueOf(batchSize));
     configProps.setProperty(AccountStatsMySqlConfig.POOL_SIZE, String.valueOf(5));
     configProps.setProperty(StatsManagerConfig.STATS_OUTPUT_FILE_PATH, localBackupFilePath.toString());
+    if (readPreviousStatsFromMysqlDb) {
+      configProps.setProperty(AccountStatsMySqlConfig.READ_PREVIOUS_STATS_FROM_MYSQL_DB, "true");
+    }
```

Comment:
you can just do
```
configProps.setProperty(AccountStatsMySqlConfig.READ_PREVIOUS_STATS_FROM_MYSQL_DB, String.valueOf(readPreviousStatsFromMysqlDb));
```

Reply:
Removed the config based on Casey's suggestion.